### ISSUE TITLE
After selecting a preset component, put focus on most commonly edited value

### ISSUE THREAD

**Issue by neilweinstock** [openrocket/openrocket#1488]
This is primarily focused on body tubes, but could apply to other components as well (centering rings, especially)

Tubes of all kinds (body tubes, inner tubes, launch lugs) are usually cut-to-length... or at least, edited-to-length in OR.  Therefore, after selecting a preset body tube, it would a small additional convenience to automatically put focus on the "length" field and select its contents, so you can immediately type in the length, rather than needing to mouse over to the field and double-click.

To me this is a no-brainer for tubes.  For other body components I'm not sure there's a good analog.

For internal components, it's "position" that is going to be edited most of the time, and that's the field that I'd pre-select.

I haven't gone through all the others to make this determination yet.

Basically, this is another way to reduce needless redundant pointing and clicking, to make it quicker and easier to get an initial design into OR.

**Comment by hcraigmiller** [comment]
Speaking as to when a preset has been selected, I agree with @neilweinstock that when selecting tubes (body tubes, tube fins, launch lugs, inner tubes, couplers, and engine blocks), the length is the best element to have for an edit default.  And, with regard to centering rings, bulkheads, and all mass components, position is likely the best element to have for an edit default.

When not using presets, I prefer the current behavior.

**Comment by neilweinstock** [comment]
Agree completely, this behavior is intended purely for preset selections.
